website/src/pages/signup.page.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react"
+import Link from "src/components/link";
+import Layout from "src/layout"
+import { centeredColumn } from "src/sprinkles.css"
+import { centeredForm, loginFormBox, positionButton, skinnyWidth, submitButton } from "./login.css";
+
+// TODO: this page is  incomplete
+const SignupPage = () => {
```

Comment:
FYI you and I hadn't talked about this together yet but we are planning on having more like an "Apply to join" flow, but that might manifest as allowing registration for a very limited privilege account until it's approved to contribute. So the registration page will need further thought but for now let's use `register` as the path to avoid "Sign in/out/up" entirely.

Reply:
Got it, updated the path.